src/components/condition-block/scss/condition-block.scss

```diff
@@ -25,7 +25,7 @@ md-dropdown {
 
 .disabled-picker {
   padding: 2px;
-  background: #c3c3c3;
+  color: var(--md-hyperlink-text-color);
```

Comment:
need to provide backup color value. make sure to look up what `--md-hyperlink-text-color` is in light mode and provide that as second parameter

Reply:
Updated